src/features/ui/MoreNFTOptions/OptionLabel/OptionLabel.tsx

```diff
@@ -0,0 +1,14 @@
+import { ReactNode } from 'react';
+
+import styles from './OptionLabel.module.scss';
+
+type OptionLabelProps = {
+	icon: ReactNode;
+	label: string;
+};
+
+export const OptionLabel = ({ icon, label }: OptionLabelProps) => (
+	<div className={styles.Container}>
+		{icon} {label}
+	</div>
+);
```

Comment:
We should add this as a component in `DropdownMenu`.

Reply:
Do you mean add `OptionLabel` to `zUI` components and then replace `item.label` with `<OptionLabel />` in the drop down component?

Reply:
Yup! `icon - text` is a common pattern for dropdown items so would be good to capture it in zUI. Not necessary for this PR though.